Calculator_Classes.rb

```diff
@@ -0,0 +1,67 @@
+=begin
+
+	Hints
+	Get user input      - ```hello = gets.chomp```
+	Output to console   - ```puts "Hello Sparta" ```  
+
+	## Tasks
+
+	Your task is to write a command line calculator application that has two settings, (b)asic and (a)dvanced. 
+
+	- First you will need to ask the user for which calculator they want to use
+	- Then you will need to ask the user for the numbers to operate on.
+	- Then you need to work out what to do with the numbers, depending on what operation they choose.
+
+	**Basic:**
+	  - Additon
+	  - Multiplication 
+	  - Division
+	  - Subtraction
+
+	**Advanced:**
+	  - Power
+	  - Square root
+=end
+
+
+
+
+class Calculator_Classes
```

Comment:
Class names should be pluralised, as they are a blueprint to making a single instance of an object. I would call this Calculator, as that is the highest level to inherit from - the basic and advanced are classes themselves that are both derivative of the parent Calculator class.

Also, class names should be in camel case e.g. CaluclatorClasses.